mmv1/products/spanner/Database.yaml

```diff
@@ -158,3 +159,8 @@ properties:
       - :GOOGLE_STANDARD_SQL
       - :POSTGRESQL
     default_from_api: true
+  - !ruby/object:Api::Type::Boolean
+    name: 'enableDropProtection'
+    default_value: false
+    description: |
+      Whether drop protection is enabled for this database. Defaults to false.
```

Comment:
Can we update the description of this field to include the distinction between this and the existing deletion_protection field?

Reply:
Done